styleguide/derek/templates/thought-leadership-overview/_partials/related-topics.ejs

```diff
@@ -0,0 +1,23 @@
+<h2 class="rsTl-relatedTopics-header">Related Topics</h2>
+<div class="rsTl-relatedTopics-row">
+  <div class="rsTl-relatedTopics-col">
+    <div class="rsTl-relatedTopics-feature">
+      <div class="rsTl-relatedTopics-img" style="background-image:url('imgs/derek/doge-small.png');"></div>
+      <div class="rsTl-relatedTopics-text">
+        <span class="rsTl-relatedTopics-type"><strong>Video</strong> (1 minute view)</span>
+        <h3 class="rsTl-relatedTopics-smallHeader">Real IT Modernization in Action</h3>
+        <a class="button lead rsTl-relatedTopics-btn">Watch the Video</a>
+      </div>
+    </div>
+  </div>
+  <div class="rsTl-relatedTopics-col">
+    <div class="rsTl-relatedTopics-feature">
+      <div class="rsTl-relatedTopics-img" style="background-image:url('imgs/derek/doge-small.png');"></div>
+      <div class="rsTl-relatedTopics-text">
+        <span class="rsTl-relatedTopics-type"><strong>Podcast</strong> (24 minute listen)</span>
+        <h3 class="rsTl-relatedTopics-smallHeader">What Real Digital Transformation Looks Like</h3>
+        <a class="button lead rsTl-relatedTopics-btn">Listen Now</a>
+      </div>
+    </div>
+  </div>
+</div> 
```

Comment:
Naturally, there is some funky behavior here in IE11/Edge we'll need to work out.